pkg/antctl/raw/traceflow/command.go

```diff
@@ -369,7 +394,7 @@ func jsonOutput(r *Response) error {
 }
 
 func getTFName(prefix string) string {
-	if !option.waiting {
+	if !option.nowait {
```

Comment:
So this is actually a fix that a nowaiting traceflow request should have random suffix to avoid conflict?

Reply:
Actually, I am not sure about the right expectation here. @gran-vmv : could you explain why just no-waiting should have a prefix?

Reply:
@lzhecheng Could you explain about this?

Reply:
I reverted my change anyway.